javalin/src/main/java/io/javalin/core/util/OptionalDependency.kt

```diff
@@ -12,6 +12,8 @@ enum class OptionalDependency(val displayName: String, val testClass: String, va
     JACKSON("Jackson", "com.fasterxml.jackson.databind.ObjectMapper", "com.fasterxml.jackson.core", "jackson-databind", "2.12.4"),
     JACKSON_KT("JacksonKt", "com.fasterxml.jackson.module.kotlin.KotlinModule", "com.fasterxml.jackson.module", "jackson-module-kotlin", "2.12.4"),
     JACKSON_JSR_310("JacksonJsr310", "com.fasterxml.jackson.datatype.jsr310.JavaTimeModule", "com.fasterxml.jackson.datatype", "jackson-datatype-jsr310", "2.12.4"),
+    JACKSON_KTORM(" Jackson KtORM","org.ktorm.jackson.KtormModule","org.ktorm","ktorm-jackson","3.4.1"),
```

Comment:
There's is an extra space in dependency name, also it's called `Ktorm`, not `KtORM` :)

Reply:
fixed, thanks for point it out!